### ISSUE TITLE
how to autoreload  in iris

### ISSUE THREAD

**Issue by 727023460** [kataras/iris#953]
Reboot service each time  too  trouble

**Comment by kataras** [comment]
Yeah, for that reason I've created the [rizla tool](https://github.com/kataras/rizla) check it out, I'll bring more features to it, like file configuration per project and etc.

**Comment by kataras** [comment]
Yeah, for that reason I've created the [rizla tool](https://github.com/kataras/rizla) check it out, I'll bring more features to it, like file configuration per project and etc.

## Install

`go get -u github.com/kataras/rizla`

## Run an Iris project

`rizla your_main.go`

**Comment by kataras** [comment]
Yeah, for that reason I've created the [rizla tool](https://github.com/kataras/rizla) check it out, I'll bring more features to it, like file configuration per project and etc.

## Install

`go get -u github.com/kataras/rizla`

## Run an Iris-powered web server

`rizla your_main.go`

**Comment by 727023460** [comment]
thanks   I have a try

**Comment by 727023460** [comment]
hi   
I use it ,
but when I change html file,page it isn't changed

**Comment by 727023460** [comment]
go program  can  hot reload

**Comment by motecshine** [comment]
`

// SetupViews loads the templates.
func (app *Application) SetupViews(viewsDir string) {
	app.RegisterView(iris.HTML(viewsDir, ".html").Reload(true))
}


`

**Comment by 727023460** [comment]
thanks

**Comment by mtatarau90** [comment]
hi guys i tried rizla but on any change the application crash 
"[INFO] 2018/07/24 15:38 A change has been detected, reloading now...
[ERRO] 2018/07/24 15:38 kill: exit status 1"

any suggestion ?

**Comment by arminschleicher** [comment]
I'm having the same issue as @mtatarau90 - any updates on this?
